addons/account/security/ir.model.access.csv

```diff
@@ -82,6 +82,7 @@ access_account_tax_internal_user,account.tax internal user,model_account_tax,bas
 access_account_tax_readonly,account.tax,model_account_tax,account.group_account_readonly,1,0,0,0
 access_account_tax_invoice,account.tax,model_account_tax,account.group_account_invoice,1,0,0,0
 access_account_tax_manager,account.tax,model_account_tax,account.group_account_manager,1,1,1,1
+access_account_tag_internal_user,account.account.tag internal user,model_account_account_tag,base.group_user,1,0,0,0
```

Comment:
I find it a bit unusual to have spaces in the name but it looks like there are already cases like that in this file.

Reply:
That field is just a human-readable name AFAIK.